content/en/docs/reference/glossary/resource.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Resource
```

Comment:
I think that we should make this term "API resource". There's already an overlap of the term resource in the context of the API and in the context of CPU and memory resources. If we're talking about an API resource in a document, especially the first time that we use the term, we should be calling it "API resource" to eliminate that ambiguity.